va/validation-authority.go

```diff
@@ -224,8 +245,25 @@ func (va ValidationAuthorityImpl) validateDvsni(identifier core.AcmeIdentifier,
 	z := sha256.Sum256(RS)
 	zName := fmt.Sprintf("%064x.acme.invalid", z)
 
+	addrs, _, _, err := va.DNSResolver.LookupHost(identifier.Value)
```

Comment:
This code looks a great deal like the similar code above. Can we create one func to be called by both sections?